README.md

```diff
@@ -0,0 +1,12 @@
+# DangerousWater
+
+Allows water damage and more realistic water functionality. Originally intended for [Mine in Abyss](https://github.com/Derongan/MineInAbyss)
+
+## Functionality
+
+* Damage users when falling into water
```

Comment:
Talk to @0ffz  a bit about this- I think there are a few intricacies worth putting in the readme

Reply:
@Derongan I might need more details on that, this section is based on my observation by looking at the source